great_expectations_provider/hooks/gx_cloud.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from airflow.hooks.base import BaseHook
+
+from great_expectations_provider.common.gx_context_actions import load_data_context
+
+
+@dataclass(frozen=True)
+class GXCloudConfig:
+    cloud_access_token: str
+    cloud_organization_id: str
+
+
+class GXCloudConnection(BaseHook):
```

Comment:
```suggestion
class GXCloudHook(BaseHook):
```
would be nice to have the suffix as Hook similar to other Airflow providers.